core/src/main/java/org/jdbi/v3/core/SqlStatement.java

```diff
@@ -1174,6 +1177,62 @@ public final This bindBySqlType(int position, Object value, int sqlType)
         return bind(position, new ObjectArgument(value, sqlType));
     }
 
+    /**
+     * Bind a parameter for each value in the given list, and defines an attribute as the comma-separated list of
+     * parameter references (using colon prefix).
+     *
+     * @param key    attribute name
+     * @param values values that will be comma-spliced into the defined attribute value.
+     * @return this
+     * @throws IllegalArgumentException if the list is empty.
+     */
+    public final This bindList(String key, List<?> values) {
```

Comment:
Should this accept `Collection` or maybe even `Iterable`?  It looks like `@BindList` supports them.